xbmc/application/Application.cpp

```diff
@@ -2321,13 +2322,12 @@ bool CApplication::PlayFile(CFileItem item, const std::string& player, bool bRes
       CUtil::ClearSubtitles();
   }
 
-  using enum CApplicationPlay::GatherPlaybackDetailsResult;
-
   CApplicationPlay appPlay{*stackHelper};
   const auto result{appPlay.GatherPlaybackDetails(item, player, bRestart)};
+  using enum CApplicationPlay::GatherPlaybackDetailsResult;
   if (result == RESULT_ERROR)
     return false;
-  else if (result == RESULT_NO_PLAYLIST_SELECTED)
```

Comment:
Why this change? Using „else“ is correct and has no downsides. With your cash future code changes here could lead to checking result twice needlessly.